presto-raptor/src/test/java/com/facebook/presto/raptor/storage/TestOrcStorageManager.java

```diff
@@ -404,7 +407,7 @@ public void testShardStatsDateTimestamp()
     public void testMaxShardRows()
             throws Exception
     {
-        OrcStorageManager manager = createOrcStorageManager(storageService, backupStore, recoveryManager, 2, new DataSize(2, MEGABYTE));
+        OrcStorageManager manager = createOrcStorageManager(storageService, backupStore, backupManager, recoveryManager, 2, new DataSize(2, MEGABYTE));
```

Comment:
Rather than in `backupManager` everywhere, just create it in `createOrcStorageManager()`?